### ISSUE TITLE
Deadlines on 1st of month could lead to anti-social practices

### ISSUE THREAD

**Issue by sjmurdoch** [ieee-security/ongoing-submission-plan#3]
I led the transition of the Privacy Enhancing Technologies Symposium (PETS) to the VLDB model and we initially had the deadline be on the 15th (every 3 months, rather than monthly). The reason was that the 1st would land on national holidays such as New Year's. The 15th seemed to be an adequate compromise.

However, this sometimes led to a weekend deadline which created an anti-social practice of weekend-working and particularly caused problems with authors with child-care responsibilities. In theory such authors could submit early, but this would put them at a disadvantage and also our submission server statistics showed this wasn't what actually happened.

For this reason, we moved to setting deadlines to avoid weekends and common holidays. Overall we found that this was beneficial and the lack of predictability was at most a minor inconvenience provided the deadlines were announced well in advance.

**Comment by parno** [comment]
That's  good point -- thanks for the observation!  We initially picked the 1st to make it very predictable and easy to remember, but I can see it creating the conflicts you noted.  We could potentially pick a year's worth of deadlines in advance, say by picking the first day of the month that is not a holiday/weekend.

**Comment by sjmurdoch** [comment]
Yes, with a little bit of preparation it should not be difficult to pick a year's worth of deadlines which meet the chosen set of constraints (avoiding holidays, weekends, even other conference deadlines if you like). I'd however suggest deadlines before holidays/weekends rather than immediately after. Otherwise people will feel pressure to work through the holiday/weekend anyway.

**Comment by enck** [comment]
One potential issue with picking the first day of the month that is not a holiday/weekend is that there are many holidays, and holidays are not uniformly observed (e.g., US vs. Europe). Predictability and simplicity has many benefits. Is there a day of the week that is less prone to holidays? For example, could you say "the first Wednesday" of every month? (I'm not sure what day would work best)

**Comment by parno** [comment]
That's a good point as well Will, and that's one of the reasons we initially proposed just picking the first of each month.  This might occasionally collide with a holiday (e.g., New Year's day), but it does avoid the need to evaluate which holidays are "important" enough to merit moving the deadline.  I'm not sure that picking the first X of the month is more or less likely to collide with a holiday than picking a particular numeric date each month.

**Comment by danwallach** [comment]
This issue is intertwined with how many deadlines you have per year. Twelve deadlines? You need consistency. Three deadlines? You can pick and choose based on the production schedule and take account of major holidays.
